packages/react/src/components/Masthead/Masthead.js

```diff
@@ -80,6 +81,31 @@ const Masthead = ({ navigation, ...mastheadProps }) => {
     });
   }, []);
 
+  const [isMastheadSticky, setSticky] = useState(false);
+  const stickyRef = useRef(null);
+  const mastheadL1Ref = useRef(null);
+
+  const mastheadSticky = cx({
+    [`${prefix}--masthead--sticky`]: isMastheadSticky,
+    [`${prefix}--masthead--sticky__l1`]: mastheadL1Ref.current != null,
+  });
+
+  useEffect(() => {
+    /**
+     * Sets sticky masthead. If bot L0 and L1 are present, L1 will be sticky.
+     * L0 will hide/show only in the top 25% of the viewport.
+     *
+     */
+    const handleScroll = root.addEventListener('scroll', () => {
+      if (mastheadL1Ref.current != null) {
+        setSticky(root.pageYOffset > root.innerHeight * 0.25);
```

Comment:
That's actually how I had it. @jeffchew used the `window-or-global` package for MastheadSearch. Basically it determines whether the component is being rendered server or client side and will adjust accordingly, since `window` wouldn't exist if it were rendered server-side.

Reply:
Yeah, cool makes sense. Thanks!